src/js/components/Rheostat/RangeRheostatGraph.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import RheostatTicker from './RheostatTicker/RheostatTicker';
+
+const RangeRheostatGraph = ({
+  resortList,
+  leftPosition,
+  rightPosition,
+  sliderMin,
+  sliderMax,
+  scoreType,
+  upperValue,
+  lowerValue,
+  graphTickerQuantity,
+}) => {
+  const tickerQuantity = Array.from(Array(graphTickerQuantity).keys());
+  const spanPercent = 100 / tickerQuantity.length;
+  const resortAverages = [];
+  const [maxResortInSpan, setMaxResortInSpan] = useState(0);
+  let height = 0;
+  let tickers = [];
+
+  // Todo: Remove filter to get average from current data,
+  //  as each resort will only have a single data point for rating type
+  resortList.forEach((resort) => {
+    let avg = 0;
+    resort.ratings.filter((rating) => rating.title === scoreType).forEach((score) => {
+      avg += score.value;
+    });
+    avg /= resort.ratings.filter((rating) => rating.title === scoreType).length;
+    resortAverages.push(avg);
+  });
+
+  // Get maxResortInSpan:
+  // Finds the most amount of resorts in a single ticker, so we can utilize full height.
+  tickerQuantity.forEach((tick) => {
+    const spanLow = spanPercent * tick;
+    const spanHigh = spanPercent * tick + spanPercent;
+
+    if (resortAverages.some((el) => el >= spanLow && el < spanHigh)) {
+      let resortQty = 0;
+
+      resortAverages.forEach((avg) => {
+        if (avg >= spanLow && avg <= spanHigh) {
+          resortQty += 1;
+        }
```

Comment:
```suggestion
      let resortQty = resortAverages.filter((avg) => avg >= spanLow && avg <= spanHigh).length;
```

I haven't run this code, but hopefully it highlights how this can be done without a loop.